packages/server/src/lib/access-helpers.js

```diff
@@ -4,6 +4,16 @@ function isPartOfAgency(agencies, agencyId) {
     return agencies.find((s) => s.id === Number(agencyId));
 }
 
+// TODO: in prod, there are a bunch of non-USDR looking users set as admin in USDR tenant?
+// some are CTG interns, but some are @cityoftulsa.org, @treasury.gov, etc.
+// probably need to move them out
+const USDR_TENANT_ID = 1;
+const USDR_AGENCY_ID = 0;
+function isUSDRSuperAdmin(user) {
+    // Note: this function assumes an augmented user object from db.getUser(), not just a raw DB row
+    return (user.tenant_id === USDR_TENANT_ID && user.agency_id === USDR_AGENCY_ID && user.role_name === 'admin');
+}
```

Comment:
Thinking more about this: maybe the right thing to do actually is to create a third `role` instead of hard-coding tenant/agency IDs like this. Would require at least:
1. Finding places that hardcode checks for admin role to also allow super-admin (including in soon-to-be-merged ARPA Reporter code)
2. Adding a filter to what roles someone can set a user to when creating or editing a user. Only super-admin can designate others as super-admin.

@bspates @mhuang-usdr thoughts?

Reply:
Punting on this for now because it adds some additional complexity (though if we ever want more than 2 roles we may end up adding such complexity anyhow). Instead, I'm restricting this "super admin" permission further by email (must be `@usdigitalresponse.org`) and filed a follow-up task #359 to clean up non-USDR users in USDR tenant and relax the email requirement.